pkg/protocol/consts/headers.go

```diff
@@ -85,4 +85,75 @@ const (
 	HTTP11 = "HTTP/1.1"
 	HTTP10 = "HTTP/1.0"
 	HTTP20 = "HTTP/2.0"
+
+	// MIME text
+	MIMETextPlain             = "text/plain"
+	MIMETextPlainUTF8         = "text/plain; charset=utf-8"
+	MIMETextPlainISO88591     = "text/plain; charset=iso-8859-1"
+	MIMETextPlainFormatFlowed = "text/plain; format=flowed"
+	MIMETextPlainDelSpaceYes  = "text/plain; delsp=yes"
+	MiMETextPlainDelSpaceNo   = "text/plain; delsp=no"
+	MIMETextHtml              = "text/html"
+	MIMETextCss               = "text/css"
+	MIMETextJavascript        = "text/javascript"
+
+	// MIME application
+	MIMEApplicationOctetStream  = "application/octet-stream"
+	MIMEApplicationFlash        = "application/x-shockwave-flash"
+	MIMEApplicationHTMLForm     = "application/x-www-form-urlencoded"
+	MIMEApplicationTar          = "application/x-tar"
+	MIMEApplicationGZip         = "application/x-gzip"
+	MIMEApplicationBZip2        = "application/x-bzip2"
+	MIMEApplicationShell        = "application/x-sh"
+	MIMEApplicationDownload     = "application/x-msdownload"
+	MIMEApplicationJavascript   = "application/javascript"
```

Comment:
Just remove it since it has been marked as deprecated by [rfc9239](https://datatracker.ietf.org/doc/html/rfc9239#name-common-javascript-media-typ)?

Reply:
ok